deploy/complete/kubernetes/mushop.yaml

```diff
@@ -1216,7 +1216,8 @@ spec:
         
       containers:
         - name: orders
-          image: "iad.ocir.io/oracle/ateam/mushop-orders:2.2.2"
+          #image: "iad.ocir.io/oracle/ateam/mushop-orders:2.2.2"
```

Comment:
There's no reason to include commented entries on the manifest. If is not using it, delete it.

Reply:
was waiting for the build version to add.

Reply:
Change the PR to WIP, so we wait to comment.